tsfc/impero.py

```diff
@@ -90,13 +90,13 @@ class Return(Terminal):
     __front__ = ('variable', 'expression')
 
     def __init__(self, variable, expression):
-        assert set(variable.free_indices) == set(expression.free_indices)
+        assert set(variable.free_indices) >= set(expression.free_indices)
```

Comment:
A constant is an expression that doesn't depend on the reference point. Yet we have to assign its to each point.